packages/icons/src/svg/processors/map.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 16"><path fill="#202020" d="M0 14a2 2 0 1 1 4 0 2 2 0 0 1-4 0Zm0-6a2 2 0 0 1 3.732-1H12c.086 0 .17.01.25.031a2 2 0 1 1 0 1.938c-.08.02-.164.031-.25.031H8.11c.04.069.078.14.114.211l1.341 2.683A2 2 0 0 0 11.355 13h.913A2 2 0 0 1 16 14a2 2 0 0 1-3.732 1h-.914a4 4 0 0 1-3.578-2.211l-1.341-2.683A2 2 0 0 0 4.645 9h-.913A2 2 0 0 1 0 8Zm0-6a2 2 0 0 1 3.75-.969C3.83 1.011 3.914 1 4 1h8c.086 0 .17.01.25.031a2 2 0 1 1 0 1.938c-.08.02-.164.031-.25.031H4a.984.984 0 0 1-.25-.031A2 2 0 0 1 0 2Z"/></svg>
```

Comment:
looks good except the fill attribute but can't remind what we said about it ?
cc @frassinier @Benrajalu

Reply:
`fill="currentcolor"` is the expected target for icons yes.

Reply:
Can we update it with a suggestion here, or does it need to be updated in Figma first?

Reply:
It should be here, Figma doesn't know `currentColor`

Reply:
Thanks @frassinier ❤️